src/Microsoft.Azure.ServiceBus/Primitives/ServiceBusConnection.cs

```diff
@@ -45,49 +44,29 @@ protected ServiceBusConnection(TimeSpan operationTimeout, RetryPolicy retryPolic
         /// </summary>
         public string SasKeyName { get; set; }
 
-        /// <summary>Gets or sets the number of messages that the message receiver can simultaneously request.</summary>
-        /// <value>The number of messages that the message receiver can simultaneously request.</value>
-        public int PrefetchCount
-        {
-            get
-            {
-                return this.prefetchCount;
-            }
-
-            set
-            {
-                if (value < 0)
-                {
-                    throw Fx.Exception.ArgumentOutOfRange(nameof(this.PrefetchCount), value, "Value must be greater than 0");
-                }
-
-                this.prefetchCount = value;
-            }
-        }
-
         internal FaultTolerantAmqpObject<AmqpConnection> ConnectionManager { get; set; }
 
         public Task CloseAsync()
         {
             return this.ConnectionManager.CloseAsync();
         }
 
-        internal MessageSender CreateMessageSender(string entityPath)
+        internal IMessageSender CreateMessageSender(string entityPath)
```

Comment:
>CreateMessageSender [](start = 32, length = 19)

Now that we are making Sender and Receiver Public and having explicit constructors to create them, we can kill this method of creating senders/receivers from here?